examples/plot_power.py

```diff
@@ -0,0 +1,76 @@
+import nipype.pipeline.engine as pe
+from nipype.interfaces.utility import IdentityInterface
+import nipype.interfaces.io as nio
+
+from ephypype.pipelines.power import create_pipeline_power
+
+from params_power import main_path, data_path
+from params_power import subject_ids, sessions
+from params_power import power_analysis_name
+from params_power import fmin, fmax, power_method, is_epoched, freq_bands
```

Comment:
@annapasca `freq_bands` was missing. You can't detect such issues unless you do continuous integration ...